src/core/unit.c

```diff
@@ -1336,6 +1337,9 @@ int unit_load(Unit *u) {
                         goto fail;
                 }
 
+                if (u->job_running_timeout != USEC_INFINITY && u->job_running_timeout > u->job_timeout)
+                        log_unit_warning(u, "JobRunningTimeoutSec= is greater than JobTimeoutSec=, the latter will override.");
```

Comment:
I don't like "override", because nothing is overridden. What about:
"JobRunningTimeoutSec= is greater than JobTimeoutSec=, it has no effect."